Lib/glyphsLib/builder/custom_params.py

```diff
@@ -322,9 +322,9 @@ def register(handler):
     # ('widthClass', 'openTypeOS2WidthClass'),
     # ('winAscent', 'openTypeOS2WinAscent'),
     # ('winDescent', 'openTypeOS2WinDescent'),
-    ("vheaVertTypoAscender", "openTypeVheaVertTypoAscender"),
```

Comment:
@belluzj Are these misspelled or did these previously work? in Glyphsapp Version 2.6.4 (1286), these names are not there.

Reply:
maybe @schriftgestalt knows?

Reply:
Ok, I've just readded them. 

If a font doesn't contain either sets of these metrics and the font is vertical, the short names will be added when converting to a ufo.

Reply:
I'm not sure. I may have assumed that anything possible on the UFO had the matching name on the Glyphs side without the openType prefix? I can't remember

Reply:
Georg posted in another comment (hidden since it's resolved) that the params with "Typo" are the long names, whilst the names without are the shorter versions. I've added both names so this shouldn't be an issue.

Reply:
https://github.com/googlefonts/glyphsLib/pull/629#discussion_r518307755